src/borg/archiver/compact_cmd.py

```diff
@@ -225,3 +233,4 @@ def build_parser_compact(self, subparsers, common_parser, mid_common_parser):
         subparser.add_argument(
             "-s", "--stats", dest="stats", action="store_true", help="print statistics (might be much slower)"
         )
+        subparser.add_argument("--iec", dest="iec", action="store_true", help="format using IEC units (1KiB = 1024B)")
```

Comment:
this is not needed: `_common.py` defines this as a common option, so it is available to all commands.